packages/components/ContextualMenu/src/ContextualMenuItem.tsx

```diff
@@ -123,8 +123,14 @@ export const ContextualMenuItem = compose<ContextualMenuItemType>({
         testID,
         ...rest,
       },
-      content: { children: text },
-      icon: createIconProps(icon),
+      content: {
+        accessible: false,
+        children: text,
+      },
+      icon: {
+        accessible: false,
+        ...createIconProps(icon),
```

Comment:
Nit: reverse the order so that `createIconProps` doesn't accidentally overwrite `accessible`